modules/core/src/constants/profile-constants.ts

```diff
@@ -29,10 +29,19 @@ export class ProfileConstants {
     private constructor() { }
 
     // SCIM2 schema IDs
+    public static readonly SCIM2_CORE_SCHEMA: string = "urn:ietf:params:scim:schemas:core:2.0";
     public static readonly SCIM2_CORE_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:core:2.0:User";
     public static readonly SCIM2_ENT_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User";
+    /**
```

Comment:
Create issue for removeing the deprecated variables in future.
https://github.com/wso2/product-is/issues/22376